canvas_modules/common-canvas/src/common-properties/controls/expression/expression-builder/expression-select-field-function.jsx

```diff
@@ -56,6 +56,9 @@ export default class ExpressionSelectFieldOrFunction extends React.Component {
 		this.fields = this._makeDatasetFields(props.controller.getDatasetMetadataFields(), props.controller.getExpressionInfo().fields);
 		this.resources = get(props.controller.getExpressionInfo(), "resources", {});
 		this.state = {
+			doubleClickedFieldRow: [],
```

Comment:
Why do we need to manage double click different from single click?

Reply:
@matthoward366 `properties-vt-row-doubleClick` class was added for styling the double clicked row(a blue border) which is not required for single click. 
For activating this class, I have used `doubleClickedFieldRow` from props. 
Couldn't rely on the doubleclick event alone, as single-clicking the `+` icon was supposed to act/look the same way as double-click